kgraphql-ktor/src/main/kotlin/com/apurebase/kgraphql/KtorFeature.kt

```diff
@@ -81,7 +81,7 @@ class GraphQL(val schema: Schema) {
                     route(config.endpoint) {
                         post {
                             val bodyAsText = call.receiveText()
-                            val request = decodeFromString(GraphqlRequest.serializer(), bodyAsText)
+                            val request = decodeFromString<GraphqlRequest>(bodyAsText)
```

Comment:
I like the previous variant more, because it fails with a compile error when I try to put in something that is not annotated as `Serializable`. I'm not sure whether this one provides the same safety as well.

Reply:
Valid point, compiles successfully without annotation. I believe I mainly tried to consolidate the different variants but will revert this.